keg/keyring.py

```diff
@@ -17,6 +17,13 @@ class KeyringError(Exception):
     pass
 
 
+class MissingValueError(Exception):
+
+    def __init__(self, key):
+        self.key = key
+        self.msg = "No value for {} found in key ring".format(key)
```

Comment:
NIT: I think the preferred way of doing this is not to set `self.msg` but to call the `super.__init__` with the message